traptor/traptor.py

```diff
@@ -216,10 +226,16 @@ def _create_birdy_stream(self):
                 self.birdy_stream = self.birdy_conn.stream.statuses.filter.post(locations=self.twitter_rules,
                                                                                 stall_warnings='true')
             except TwitterApiError as e:
-                self.logger.critical(e)
+                self.logger.critical("Caught Twitter Api Error", extra={
+                    'error_type': 'TwitterAPIError',
+                    'ex': traceback.format_exc()
+                })
                 sys.exit(3)
         else:
-            self.logger.critical('That traptor type has not been implemented')
+            self.logger.critical('Caught error creating birdy stream for Traptor type that does not exist', extra ={
+                'error_type': 'NotImplementedError',
+                'ex': traceback.format_exc()
```

Comment:
As this is a critical error that will cause Traptor to not function, perhaps we should force raise the Not Implemented Error or another? I'm not sure if that will give us the traceback, or if it's actually necessary in this case. I'm thinking the main thing we want to know is that someone is attempting to fire up an unsupported type of Traptor.

@jasonrhaas any alternative suggestions?